### PR TITLE
Fixed 'public: false' in services

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: src/Bridge/Symfony/Resources/config/services.yml [hunk 1]
```diff
@@ -10,23 +10,23 @@ services:
 
     seo_override.subscriber:
         class: Joli\SeoOverride\Bridge\Symfony\Subscriber\SeoSubscriber
-        private: true
+        public: false
```
Comment (pyrech) [inline src/Bridge/Symfony/Resources/config/services.yml]: The event subscriber must be public (cf the build error :smile:)

Comment (ternel) [inline src/Bridge/Symfony/Resources/config/services.yml]: 😭

**Comment by lyrixx** [comment]
Here we go, all green ;)

**Comment by pyrech** [comment]
Thanks :+1:

**Pull request merged**